anymail/backends/sendinblue.py

```diff
@@ -0,0 +1,144 @@
+from .base_requests import AnymailRequestsBackend, RequestsPayload
+from ..exceptions import AnymailRequestsAPIError
+from ..utils import get_anymail_setting, parse_address_list
+
+
+class EmailBackend(AnymailRequestsBackend):
+    """
+    SendinBlue v3 API Email Backend
+    """
+
+    esp_name = "SendinBlue"
+
+    def __init__(self, **kwargs):
+        """Init options from Django settings"""
+        esp_name = self.esp_name
+        self.api_key = get_anymail_setting(
+            'api_key',
+            esp_name=esp_name,
+            kwargs=kwargs,
+            allow_bare=True,
+        )
+        api_url = get_anymail_setting(
+            'api_url',
+            esp_name=esp_name,
+            kwargs=kwargs,
+            default="https://api.sendinblue.com/v3",
+        )
+        if not api_url.endswith("/"):
+            api_url += "/"
+        super(EmailBackend, self).__init__(api_url, **kwargs)
+
+    def build_message_payload(self, message, defaults):
+        return SendinBluePayload(message, defaults, self)
+
+    def raise_for_status(self, response, payload, message):
+        if response.status_code < 200 or response.status_code >= 300:
+            raise AnymailRequestsAPIError(
+                email_message=message,
+                payload=payload,
+                response=response,
+                backend=self,
+            )
+
+    def parse_recipient_status(self, response, payload, message):
+        # SendinBlue doesn't give any detail on a success
+        # https://developers.sendinblue.com/docs/responses
+
+        return dict()
```

Comment:
This *must* return a dict mapping recipient email address (addr_spec) to an AnymailRecipientStatus for each message recipient. Since the best we know here is "probably sent", you could use [code like this](https://github.com/anymail/django-anymail/blob/v1.4/anymail/backends/mailgun.py#L54-L56) to mark all recipients "queued".

The AnymailRecipientStatus.message_id is some sort of tracking ID that can be used to correlate this send with later webhook status events. It's not documented, but a quick test suggests SiB's response is actually json, and *does* include a usable tracking id: `{"messageId":"<201802182116.25729519001@smtp-relay.mailin.fr>"}`.

Reply:
Exactly responses i've on my tests look like this one :
```
{
    'code': 'success',
    'message': 'Email sent successfully',
    'data': {
        'message-id': '<201802182329.73817096184@smtp-relay.sendinblue.com>'
    }
}
```

Thanks for the link on the mailgun example. It was one of my point on my todo-list of research 👍